superset-frontend/src/filters/components/Select/controlPanel.ts

```diff
@@ -35,17 +29,21 @@ const config: ControlPanelConfig = {
     {
       label: t('Query'),
       expanded: true,
+      controlSetRows: [['groupby']],
+    },
+    {
+      label: t('UI Configuration'),
+      expanded: true,
       controlSetRows: [
-        ['groupby'],
-        ['metrics'],
-        ['adhoc_filters'],
         [
           {
             name: 'multiSelect',
             config: {
               type: 'CheckboxControl',
               label: t('Multiple select'),
               default: multiSelect,
+              resetConfig: true,
+              renderTrigger: true,
```

Comment:
Filtering on only renderTrigger might be fragile. But we can probably come up with a better option for this in a follow-up PR.

Reply:
done